Application/src/model/Menu.java

```diff
@@ -37,6 +56,11 @@
     private final static String text_auto_pause = " || ";
     private final static String text_next = "->";
     private final static String text_previous = "<-";
+    private final static String text_import_lobby = "Import to lobby";
+    private final static String text_join_lobby = "Join a lobby";
```

Comment:
Why is this the only one that uses 'a' join A lobby? Please use the same standard for all of them.